client/app/components/subscription-form.js

```diff
@@ -1,44 +1,34 @@
-import Component from '@ember/component';
-import EmberObject, { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { action, set } from '@ember/object';
 import { lookupCommunityDistrict } from '../helpers/lookup-community-district';
 import { tracked } from '@glimmer/tracking';
 
 export default class SubscriptionFormComponent extends Component {
-    subs = EmberObject.create({
-        'brooklyn': []
-    });
+
+    communityDistrictsByBorough = {};
+    isCommunityDistrict = false;
     constructor(...args) {
         super(...args);
 
-        // this.set('subs', {
-        //     'brooklyn': []
-        // })
-        const lookupCommunityDistrictObj = lookupCommunityDistrict();
-        for (let i = 0; i < lookupCommunityDistrictObj.length; i += 1) {
-            const district = lookupCommunityDistrictObj[i];
-            if (district.boro == 'Brooklyn') {
-                this.subs['brooklyn'].push({ ...district, 'checked': 0 });
+        const districts = lookupCommunityDistrict();
+        for (const district of districts) {
```

Comment:
It would be more in line with previous usage and more performant if you put this in the helper.